bundles/org.openhab.binding.dlinksmarthome/src/main/java/org/openhab/binding/dlinksmarthome/internal/DLinkSmartHomeDiscoveryParticipant.java

```diff
@@ -62,8 +73,17 @@ public DiscoveryResult createResult(final ServiceInfo serviceInfo) {
 
         final ThingTypeUID thingTypeUID = getThingType(serviceInfo);
 
-        if (THING_TYPE_DCHS150.equals(thingTypeUID)) {
-            return createMotionSensor(thingUID, thingTypeUID, serviceInfo);
```

Comment:
Can you add a comment to the PR description why you removed this device?

Reply:
It's not removed. It's just moved to the set part of `SUPPORTED_THING_TYPES_UIDS`